arch/common/gen_isr_tables.py

```diff
@@ -227,6 +227,9 @@ def main():
     nvec = intlist["num_vectors"]
     offset = intlist["offset"]
 
+    if nvec > 2 ** 15:
+        raise ValueError('nvec is too large, check endianess.')
```

Comment:
correct spelling is ``endianness``

Reply:
Fixed.

I used `build` as the subsystem for the commit message. Is this correct?

Reply:
@hackaugusto, I would use "arch: common: "